GSA-TTS/gitlab-runner-cloudgov
# Integration Test for Go CF Driver

# 🎫 Addresses issue: https://github.com/GSA-TTS/devtools-program/issues/198


## 🛠 Summary of changes


Adds an integration test that verifies the cf-driver-go is able to connect to cloud.gov and get an app list.


## 📜 Testing Plan

How would a peer test this work?

- You would need to first get a username & password for some space on cloud.gov that has at least one app.
- Then you add those credentials to `./runner/cf-driver-go/cg/testdata/.cg_creds` in the style of the `.sample` there.
- Run `make integration`, which should give you an error and tell you what the `[]Apps` JSON looks like.
- Copy that JSON over the last line for your testdata file and run `make integration` again, this time it should succeed.
